.gitbook/0-about-panacea/1-data-exchange-protocol/1-DEP-problems-it-solves.md

```diff
@@ -0,0 +1,51 @@
+# DEP and the problems it solves
+
+DEP seeks to address the problems of decentralized data delivery that allows for data validation while ensuring data privacy:
+
+- [Data privacy](#data-privacy)
+- [Data validation](#data-validation)
+- [Incentive](#incentive)
+- [Decentralization](#decentralization)
+- [Scalability](#scalability)
+
+## Data privacy
+
+Since DEP is targeting sensitive data, the privacy of the data must be ensured throughout all data transfers. 
+It is essential that no data is exposed in the process of data delivery.
+
+DEP uses data encryption in all data delivery processes to ensure the privacy of sensitive data. 
+Symmetric and asymmetric key encryption is utilized to ensure that the privacy of data during all data delivery processes.
+
+## Data validation
+
+As a data consumer, it is important to validate that the data you receive is the right data.
+The data must be truthful, uncorrupted, and conform to the requirements the consumer propose.
+
+DEP provides a way to verify data by utilizing oracle running in an enclave environment, did-based verifiable presentation.
+
+## Incentive
+
+By providing data, the data provider can be guaranteed with an incentive offered by the data consumer. 
+In this case, the delivery of data and the paying incentive must be processed atomically and simultaneously. 
+
+DEP ensures this through the panacea blockchain Tx and data encryption.
+
+## Decentralization
+
+DEP aims to be a decentralized protocol with no centralized data validator and deliverer.
+
+The DEP components, panacea blockchain and oracle, are open decentralized ecosystem where everyone can participate.
```

Comment:
I think this part is quite weak. If you have any suggestions, please comments.

Reply:
```suggestion
## Decentralization

DEP aims to be a decentralized protocol with no centralized data validator and deliverer.

Anyone can participate in operating DEP components, such as Panacea blockchain and oracle. By the nature of the proof-of-stake blockchain, any party with less than 2/3 voting power cannot change the behavior of DEP, and anyone with less than 1/3 voting power cannot stop the protocol. Also, anyone can operate oracle nodes to improve the performance of DEP and earn incentives if they have appropriate hardware environments.
The entire source code, protocol specifications, and discussions are open-sourced, so anyone can dive into details and participate in developments. Any protocol changes must go through governments to be deployed in production. 
```

Reply:
Thank you. I think this is clearer.